docs/collision_avoidance_tutorial.md

```diff
@@ -194,6 +201,19 @@ collision_arm_table:
   py_class_name: 'CollisionArmTable'
   py_module_name: 'collision_arm_table'
 
+```
+However if we create our custom class within the 'stretch_body' directoy the yaml file should be configured as follows:
+
+```yaml
+robot_collision:
+  models:
+  - collision_arm_table
+
+collision_arm_table:
+  enabled: 1
+  py_class_name: 'CollisionArmTable'
+  py_module_name: 'stretch_body.collision_arm_table'
```

Comment:
Let's consolidate these two. I think we should recommend they do not create the file in the stretch_body directory (so the export command becomes necessary).

Reply:
Will do. I will remove this block and the text above and just include the explanation of the bash command.